code/prisonersDilemma.py

```diff
@@ -30,8 +30,10 @@ def getVisibleHistory(history, player, turn):
 
 def strategyMove(move):
     if type(move) is str:
-        defects = ["defect","tell truth"]
+        defects = ["defect","tell truth", "D"]
         return 0 if (move in defects) else 1
+    elif type(move) is bool:

```

Comment:
This check isn't necessary since bools are converted into 0 or 1 by the `else` anyway. (And also the numpy array type is `int`, so it will be converted to an int there.